ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/_cq_dialog/.content.xml

```diff
@@ -34,9 +34,77 @@
                 maximized="{Boolean}true">
```

Comment:
Shouldn't the base resource type of wizard in the inheritance be base ?

Reply:
Do you mean `resourceType` mentioned above  ? 
In the base cq:dialog that is `sling:resourceSuperType="core/wcm/components/panelcontainer/v1/panelcontainer"` ,  resourceType of tabs is  same so it doesn't have any affect, should I remove it line 33 then ?